File: code/local-development/airdropping-sol/airdropping-sol.preview.en.ts
const airdropTestSOL = async() => {
  try {
    const connection = new Connection(clusterApiUrl("devnet"), "confirmed");
    const airdropSignature = await connection.requestAirdrop)
      keypair.publicKey,
      LAMPORTS_PER_SOL
    );
    
    const latestBlockHash = await connection.getLatestBlockhash();
    
    await connection.confirmTransaction({
        blockhash: latestBlockHash.blockhash,
        lastValidBlockHeight: latestBlockHash.lastValidBlockHeight,
        signature: airdropSignature,
    });
  } catch (error) {
    console.log(error);
  };
};

// The following code blocks are deprecated.
// const airdropSignature = await connection.requestAirdrop(
//   keypair.publicKey,
//   LAMPORTS_PER_SOL
// );

// await connection.confirmTransaction(airdropSignature);

File: code/local-development/airdropping-sol/airdropping-sol.preview.en.rs
match client.request_airdrop(&pubkey, LAMPORTS_PER_SOL) {
    Ok(sig) => loop {
        if let Ok(confirmed) = client.confirm_transaction(&sig) {
            if confirmed {
                println!("Transaction: {} Status: {}", sig, confirmed);
                break;
            }
        }
    },
    Err(_) => println!("Error requesting airdrop"),
};